packages/patterns/src/keys/keycollection-operators.js

```diff
@@ -21,21 +21,14 @@ const { quote: q, Fail } = assert;
  * rank, into an iterable that resolves those ties using `fullCompare`.
  *
  * @template [V=unknown]
- * @param {Iterable<[Key, V]>} rankSorted
+ * @param {Array<[Key, V]>} entries
```

Comment:
No objection, but do you like this better than

```suggestion
 * @param {[Key, V][]} entries
```
? I always use the bracket notation rather than `Array<...>`

Reply:
I like the generic notation better, especially for cases like this where we waffled between Array and Iterable (the latter of which has no shorthand notation AFAIK). But I will sometimes use `NamedType[]` for brevity.